src/main/java/com/tabnine/binary/fetch/BootstrapperSupport.java

```diff
@@ -0,0 +1,73 @@
+package com.tabnine.binary.fetch;
+
+import com.intellij.openapi.diagnostic.Logger;
+import com.intellij.util.text.SemVer;
+import com.tabnine.general.StaticConfig;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.prefs.Preferences;
+
+import static java.lang.String.format;
+
+public class BootstrapperSupport {
```

Comment:
Nothing here needs to be static

Reply:
its a piure function. I don't want to create an instance for this